docs/release/trg-0/trg-1-06.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TRG 1.06 - Administrator's Guide (User Assistance)
+sidebar_position: 1
+---
+
+:::caution
+Proposed release date: 15th of January 2024
+:::
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 31-Jan-2024  | created initial Draft                  |
+
+## Why
+
+Administrator's Guide is intended for administrators that install and operate the solution.
+
+## Description
+
+- The Admin Guide **has to** be stored in the folder: ```docs/admin/```
+- Admin Guide is available with the software at the same time.
+- Admin Guide is correct and up to date; English is a must.
```

Comment:
I think correctness is too hard to check, so I would not mention it in the TRG and trust in the contributors and committers, that they describe everything to the best of their knowledge